aireview/README.md

```diff
@@ -1,2 +1,54 @@
 # AIReview
-Pull request review agent
\ No newline at end of file
+Pull request review agent
```

Comment:
**documentation**: Missing step for installing dependencies. Add specific command `pip install -r requirements.txt` or list of required packages.